methcomp/blandaltman.py

```diff
@@ -115,76 +123,44 @@ def plot(self, ax: matplotlib.axes.Axes):
         ax.spines['top'].set_visible(False)
 
         # set X and Y limits
-        if self.xlim is not None:
-            ax.set_xlim(self.xlim[0], self.xlim[1])
-        if self.ylim is not None:
-            ax.set_ylim(self.ylim[0], self.ylim[1])
+        if xlim is not None:
+            ax.set_xlim(xlim[0], xlim[1])
+        if ylim is not None:
+            ax.set_ylim(ylim[0], ylim[1])
 
         # graph labels
-        ax.set_ylabel(self.y_title)
-        ax.set_xlabel(self.x_title)
-        if self.graph_title is not None:
-            ax.set_title(self.graph_title)
+        ax.set_ylabel(y_title)
+        ax.set_xlabel(x_title)
+        if graph_title is not None:
+            ax.set_title(graph_title)
 
 
 def blandaltman(method1, method2,
-                x_label='Mean of methods', y_label='Difference between methods', title=None,
-                diff='absolute', limit_of_agreement=1.96, reference=False, CI=0.95,
-                xlim=None, ylim=None,
-                color_mean='#008bff', color_loa='#FF7000', color_points='#000000',
-                point_kws=None,
-                ax=None):
-    """Provide a method comparison using Bland-Altman plotting.
+                diff='absolute', limit_of_agreement=1.96, CI=0.95) -> _BlandAltman:
```

Comment:
To type annotate the version with optional tuple return value
```python
from typing import Union, Tuple
...
-> Union[Tuple[matplotlib.axes.Axes, _BlandAltman], matplotlib.axes.Axes]
```

Reply:
we will return a class isntance now, right?

Reply:
Yep, `-> BlandAltman`